paasta_tools/cli/cmds/check.py

```diff
@@ -139,7 +139,7 @@ def makefile_has_a_tab(makefile_path):
 
 def makefile_has_docker_tag(makefile_path):
     contents = get_file_contents(makefile_path)
-    return re.search(r"^DOCKER_TAG\s*\?=", contents, re.MULTILINE) is not None
+    return re.search(r"DOCKER_TAG\s*\?=", contents, re.MULTILINE) is not None
```

Comment:
To be honest, we'll never get this 100% right, which is why `paasta check` is only advisory. I don't care too much to try and make the "perfect regex".

But actually, `?=` is pretty important - paasta needs the makefile to respect DOCKER_TAG if it's already set.